nussknacker-dist/src/universal/conf/docker-compose-application.conf

```diff
@@ -1,5 +1,5 @@
 db {
```

Comment:
this file certainly shouldn't be in our dist package

Reply:
I'v moved it to demo/docker/app